duckduckgo/macos-browser
# fix(duck-player): capture the correct url before a click occurs

Task/Issue URL: https://app.asana.com/0/0/1204843414383563/f
Tech Design URL:
CC:

**Description**:

**Steps to test this PR**:
1. set `always open in duck player`
2. click a thumbnail on https://www.youtube.com
3. it should load correctly in duck player



---
###### Internal references:
[Pull Request Review Checklist](https://app.asana.com/0/1202500774821704/1203764234894239/f)
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
[Pull Request Documentation](https://app.asana.com/0/1202500774821704/1204012835277482/f)
